src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto

```diff
@@ -61,6 +63,26 @@ message PaymentDetailsRequest {
   }
 }
 
+message GetPaymentTokenRequest {
+  string store_id = 1; // Your Kody store id
+  oneof payment_identifier {
+    string payment_id = 2; // The unique identifier created by Kody
+    string payment_reference = 3; // Your unique payment reference that was set during the initiation
+  }
+}
+
+message GetPaymentTokenResponse {
+  string token = 1;
+  string expiry = 2;
+  IntegratedPaymentTokenStatus status = 3;
+
+  enum IntegratedPaymentTokenStatus {
```

Comment:
Does it need to be specifically stated that it is an IntegratedPaymentToken? Couldn't this be generic for any type of PaymentMethodToken?

Reply:
Also it might be useful to review our naming for the Token. It is not a payment token.
It would be preferable / more accurate to be one of the below:
• Card token
• Payment method token
• PAN token
• Account token

Reply:
I see payment method token and payment token as being quite similar. Choose a name though and I'll be happy to go with it.

Reply:
My concern of Payment Token is that it might be perceived as: a token for this payment (another identifier for this payment).
While the Payment Method Token represents the proxy identifier for the payment method (the card).

To keep things simpler, I would just call it: Card Token, but if we are pedantic enough, the correct one would be PAN Token.
https://gocardless.com/guides/posts/primary-account-number-pan-definition/

Reply:
Done 👍